profiles/foot.lua

```diff
@@ -160,6 +160,9 @@ function way_function (way, result)
   if "roundabout" == junction then
     result.roundabout = true
   end
+  if "circular" == junction then
+    result.circular = false
```

Comment:
well... I'd argue that foot should neither announce roundabout nor circular. After all: what is an exit here? Sidewalk?